src/modules/avformat/producer_avformat.c

```diff
@@ -132,11 +132,12 @@ struct producer_avformat_s
 	int is_audio_synchronizing;
 	int video_send_result;
 #if USE_HWACCEL
-	int hw_pix_fmt;
-	int hw_device_type;
-	char hw_device[128];
-	AVBufferRef* hw_device_ctx;
-	AVFrame* sw_video_frame;
+	struct {
+		int pix_fmt;
+		int device_type;
+		char device[128];
+		AVBufferRef* device_ctx;
+	} hwaccel;
```

Comment:
This change is purely cosmetic. Although I personally thinks this looks nicer since it better isolates the hwaccel context, changing it back is not a problem.